TestConnectionPings: server: failed to send any pings (check logs)

### Which jobs are flaking?

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/116127/pull-kubernetes-unit/1630794267716751360



### Which tests are flaking?

TestConnectionPings

### Since when has it been flaking?

Not sure

### Testgrid link

pull-kubernetes-unit

### Reason for failure (if possible)

```
k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/httpstream/spdy: TestConnectionPings expand_less | 0s
-- | --
{Failed  === RUN   TestConnectionPings     connection_test.go:240: server: failed to send any pings (check logs) --- FAIL: TestConnectionPings (0.04s) }

```

### Anything else we need to know?

https://github.com/kubernetes/kubernetes/blob/a023a5eb17471410ec0b07cd6c78bdd356820875/staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go#L185

### Relevant SIG(s)

/sig api-machinery

--- a/staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go
+++ b/staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go
@@ -247,12 +247,12 @@ func TestConnectionPings(t *testing.T) {
 		t.Fatalf("client: error connecting to proxy: %v", err)
 	}
 	defer clConn.Close()
-	start := time.Now()
 	clSPDYConn, err := NewClientConnection(clConn)
 	if err != nil {
 		t.Fatalf("client: error creating spdy connection: %v", err)
 	}
 	defer clSPDYConn.Close()
+	start := time.Now()
 	clSPDYStream, err := clSPDYConn.CreateStream(http.Header{})
 	if err != nil {
 		t.Fatalf("client: error creating stream: %v", err)

